### DISCUSSION TITLE
Sudden User Logouts and 'Restricted' Message Issue with Supabase

### DISCUSSION THREAD

**Discussion by joemar25** [supabase/supabase#29655] [Questions] [answered]
Many users who are logging into our mobile app using Supabase are suddenly being logged out. After logging in again, they are seeing a 'restricted' message. Has anyone else experienced this issue or found a solution?

**Comment by GaryAustin1** [comment, marked as answer]
Maybe this:  https://status.supabase.com/
Also: https://github.com/orgs/supabase/discussions/29370

If the first I assume it has resolved for you by now.
Also contact support or provide more info if still an issue.

**Reply by joemar25** [reply]
Thanks
